### BIG differences, this is diff of wrong files. Both are hard to find.
### +++ CLUS
### --- CLHS

--- saved-pages-dpans3/cl:functions:math-add.txt	2017-04-27 21:53:02.000000000 +0100
+++ saved-pages-clus/cl:functions:math-add.txt	2017-04-27 21:53:02.000000000 +0100
@@ -1,2 +1,32 @@
-Please select which reference to + you intended
+Function
-FunctionVariable
\ No newline at end of file
+Syntax
+ + &rest numbers → sum
+Arguments and Values
+ number - a number.
+ sum - a number.
+Description
+Returns the sum of numbers, performing any necessary type conversions in the process. If no numbers are supplied, 0 is returned.
+Examples
+(+)
+→
+0
+(+ 1)
+→
+1
+(+ 31/100 69/100)
+→
+1
+(+ 1/5 0.8)
+→
+1.0
+Affected By
+None.
+Exceptional Situations
+Might signal type-error if some argument is not a number. Might signal arithmetic-error.
+See Also
+ {\secref\NumericOperations}
+ {\secref\RationalComputations}
+ {\secref\FloatingPointComputations}
+ {\secref\ComplexComputations}
+Notes
+None.
\ No newline at end of file
